src/test/resources/features/createCampaign.feature

```diff
@@ -6,6 +6,7 @@ Feature: Create Campaign
     When I create a new campaign with required fields
       | Campaign Name           | New Campaign |
     Then I verify Campaign created with requirement fields
+    And I verify date create
```

Comment:
This step should be more descriptive such as: I verify that the date matches the creation date, or I verify that the date detail matches the current day and hour.

Reply:
good suggestion